modules/RTC/RTCUtils.js

```diff
@@ -864,19 +866,15 @@ class RTCUtils extends Listenable {
                     };
                 }
             } else if (RTCBrowserType.isEdge()) {
-                // TODO: Remove when EDGE is fully supported. For now ensure
-                // that, if EDGE is detected, it's just unsupported.
-                if (RTCBrowserType.isEdge()) {
+                // Disable until fully implemented.
+                if (!ENABLE_EDGE) {
                     rejectWithWebRTCNotSupported(
-                        'Microsoft EDGE not yet supported', reject);
+                        'Microsoft Edge not yet supported', reject);
 
                     return;
                 }
 
-                // TODO: Uncomment when done. For now use the Edge native
-                // RTCPeerConnection.
-                // this.peerconnection = ortcRTCPeerConnection;
-                this.peerconnection = RTCPeerConnection;
+                this.peerconnection = ortcRTCPeerConnection;
```

Comment:
I've tried to rebase into a separate forked branch (`git pull --rebase mainstream master`)  and things are broken:

* My changes to `RTCUtils.js` (such as the new ENABLE_EDGE bool) are dropped.
* In `TraceablePeerConnection.js` I **always** get this fu****** error:

```js
<<<<<<< 296134ab3edd899f0a65395b678614f7b9ace979
import RTCBrowserType from './RTCBrowserType';
=======
import RTCBrowserType from './RTCBrowserType.js';
import ortcRTCPeerConnection from './ortc/RTCPeerConnection.js';
>>>>>>> Working on ortc.RTCPeerConnection shim
```

No idea how to get rid of that error! it always happens when I rebase!